### DISCUSSION TITLE
URL in PR to old repository

### DISCUSSION THREAD

**Discussion by jgelon** [renovatebot/renovate#15558] [Request Help]
Renovate is currently working in our Azure DevOps environment and we love the updates. We are however facing a minor issue; in the PR the url to the Repository is added as a hyperlink. We renamed that repository but Renovate is still linking to the old repo.

Example:
Repo old: CompanyExtensions (http://mainurl/repos/OLD)
Repo new name: CompanyExtensions (http://mainurl/repos/NEW)

Text in Pull Request:
Package | Type | Update | Change
-- | -- | -- | --
CompanyExtensions  (links to http://mainurl/repos/OLD)  | nuget | minor | 2.60.0 -> 2.63.0

Is this a caching issue (and if so, how to reset) or something else (and how to fix it)?

**Comment by rarkins** [comment]
Are you using a v2 or v3 NuGet registry?

**Reply by rarkins** [reply]
Here is how the v3 datasource determines the sourceUrl: https://github.com/renovatebot/renovate/blob/d50856bda75fb72ba9ccf32565747d00e8649113/lib/modules/datasource/nuget/v3.ts#L175
